src/provider/rpc.ts

```diff
@@ -149,48 +158,50 @@ export class RpcProvider implements ProviderInterface {
   ): Promise<BigNumberish> {
     const parsedKey = toHex(toBN(key));
     const block = new Block(blockIdentifier);
-    return this.fetchEndpoint('starknet_getStorageAt', [
-      contractAddress,
-      parsedKey,
-      block.identifier,
-    ]);
+    return this.fetchEndpoint('starknet_getStorageAt', {
+      contract_address: contractAddress,
+      key: parsedKey,
+      block_id: block.identifier,
```

Comment:
nitpick: on line https://github.com/0xs34n/starknet.js/pull/332/files#diff-6ae3f9177722bfbaef720fe10fadc424e40d7895481d9a7c09aac78cccf3b17fR134 we have the `block_id` instead of `block.identifier`, so stylistically (consistency) maybe to use only one or the other

Reply:
https://github.com/0xs34n/starknet.js/pull/332/commits/24c24e8060b76f6a300b21c3148a15183b802fcb